src/Microsoft.IdentityModel.Protocols.OpenIdConnect/OpenIdConnectProtocolValidationContext.cs

```diff
@@ -59,8 +53,13 @@ public class OpenIdConnectProtocolValidationContext
         public string State { get; set; }
 
         /// <summary>
-        /// Gets or sets a validated id_token.
+        /// Gets or sets the response received from userinfo_endpoint.
+        /// </summary>
+        public JwtSecurityToken UserInfoEndpointResponse { get; set; }
+
+        /// <summary>
+        /// Partially validated id_token. This has gone through basic validation - 'aud', 'iss', 'lifetime' and 'signature'.
```

Comment:
Might want to start the comment with: Cryptographically validated id_token. the term 'basic validation' doesn't add much as the term is not defined anywhere. Perhaps just list what was validated: signature, audience, issuer, lifetime.